addons/website_event_track/static/src/js/service_worker.js

```diff
@@ -236,14 +236,19 @@ const matchCache = async (request) => {
 
 /**
  *
- * @param {FetchEvent} param0
+ * @param {Request} request
+ * @param {Boolean} doNotTrack whether adding a header preventing the server to track the request
  * @returns {Promise<Response>}
  */
-const processFetchRequest = async ({ request }) => {
+const processFetchRequest = async (request, doNotTrack) => {
     const requestCopy = request.clone();
     let response;
     try {
-        response = await fetch(request);
+        if (doNotTrack) {
```

Comment:
Do we want to track at all in the service worker?

We could say that we only track regular page accesses and simplify it here by always adding the disable tracking header.

Reply:
That would be nice indeed. What do you think @pparidans ? If I have well understood your previous message, the requests in processPendingRequests are real call from the user (replayed) but you also mentioned that it concerns only one URL.

Reply:
> Do we want to track at all in the service worker?
> We could say that we only track regular page accesses and simplify it here by always adding the disable tracking header.
Bear in mind that once you have a service worker which listen to "fetch" event, **all** requests pass through it! So disabling tracking for all fetch request would result in no tracking at all (at least under `/event` in this case).